HackYourFuture-CPH/Guide-IT
# added email registration form component with storybook

# Description

Added form with email input and button and when user clicks without entering email or if enters an invalid email, user will get an 
error message and warning icon 

Fixes # (issue)
#40 

# How to test?
using npm run storybook and **To check error message and sign give a wrong email by entering wrongly .com i.e .om or .cm...**
# Checklist
- [✓] I have performed a self-review of my own code
- []✓ I have commented my code, particularly in hard-to-understand areas
- [✓] I have made corresponding changes to the documentation
- [✓] This PR is ready to be merged and not breaking any other functionality
